docs/REST API/rest-api.md

```diff
@@ -5489,6 +5490,67 @@ Returns the query specified by ID.
 }
 ```
 
+### Get query report
+
+Returns the query report specified by ID.
+
+`GET /api/v1/fleet/queries/{id}/report`
+
+#### Parameters
+
+| Name | Type    | In   | Description                                |
+| ---- | ------- | ---- | ------------------------------------------ |
+| id   | integer | path | **Required**. The id of the desired query. |
+| order_key       | string  | query | What to order results by. Can be any column in the queries table.                                                             |
+| order_direction | string  | query | **Requires `order_key`**. The direction of the order given the order key. Options include `asc` and `desc`. Default is `asc`. |
+
+#### Example
+
+`GET /api/v1/fleet/queries/31/report`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "query_id": 31,
+  "results": [
```

Comment:
Tentative change:

A. Adding "errors" list, or either:
B. Add "columns" to results:
```go
{ 
 host_id: X,
 columns: {
      "model": "USB 2.0 Hub",
      "vendor": "VIA Labs, Inc."
  }
}
{
  host_id: X,
  columns: ...
}
{ 
  host_id: Z,
  error: "asdasdasd",
}
```
(To solve "what if a scheduled query is `SELECT error, osquery_version FROM mytable;`" )